webots_ros2_core/webots_ros2_core/webots_differential_drive_node.py

```diff
@@ -58,43 +56,40 @@ def __init__(self,
         self.right_motor.setPosition(float('inf'))
         self.left_motor.setVelocity(0)
         self.right_motor.setVelocity(0)
-        self.create_subscription(Twist, command_topic,
-                                 self._cmd_vel_callback, 1)
+        self.create_subscription(Twist, command_topic, self._cmd_vel_callback, 1)
 
         # Initialize odometry
         self.reset_odometry()
         self.left_wheel_sensor = self.robot.getPositionSensor(left_encoder)
         self.right_wheel_sensor = self.robot.getPositionSensor(right_encoder)
         self.left_wheel_sensor.enable(self.timestep)
         self.right_wheel_sensor.enable(self.timestep)
-        self._odometry_publisher = self.create_publisher(
-            Odometry, odometry_topic, 1)
+        self._odometry_publisher = self.create_publisher(Odometry, odometry_topic, 1)
         self._tf_broadcaster = TransformBroadcaster(self)
 
         # Initialize timer
-        self.create_timer(self.timestep / 1000.0, self._publish_odometry_data)
+        self._last_sample_time = self.robot.getTime()
```

Comment:
I would add the `odometer` keyword in this variable so that we know what this sample time refer to.